cmd/server.go

```diff
@@ -129,6 +134,17 @@ func main() {
 		bufferManager = &buffer.OnDemandBufferManager{}
 	}
 
+	var tileCache cache.Cache
+	if redisAddr != "" {
+		client := redis.NewClient(&redis.Options{
+			Addr: redisAddr,
+		})
+		tileCache = cache.NewRedisCache(client)
```

Comment:
any sort of error checking worth doing here?  Seems weird to just presume that everything worked.  I'll keep reading

Reply:
Yea, I could add a `client.Ping()` to make sure the connection worked before moving on.

Reply:
Fixed in 32942de.